### ISSUE TITLE
2022 facelift for Ruby.sg

### ISSUE THREAD

**Issue by TayKangSheng** [rubysg/rubysg-reboot#346]
https://ruby.sg is the face of the RubySG group on the internet. We should keep it updated and not let it feel like it's a dead page on the internet. I'm proposing that we do a simple redesign of the site, and take this opportunity to ✂️ things that are no longer relevant to the RubySG group today. Adding new feature will be out of scope for now as to keep the facelift simple and can be implemented quickly.

### tldr

1. New design for Ruby.sg
2. Things that we consider cutting 
    * links / refs to RDRC since it's not going to come back anytime soon. 
    * clean up the `Join Us` section. At the moment, telegram group is the only active surface so we can consider cutting the rest.

**Comment by TayKangSheng** [comment]
# Meeting - 3rd March

Attendees: Kang Sheng, Nicholas, Charmaine

## Notes

### New design doesn't mean like we have to overhaul the entire thing. 
* keep the red and white design
* some update to the design language to "feel" more refreshed / modern

### Outdated things that we can cut 
* Photos. Photos are outdated, and we don't have many photos because we have moved to online meetups for a while already. In exchange, maybe we can do something else that can represent that there is an active community around it. Like crowdsource github / twitter avatars etc. 
* Red dot ruby conf. the last conference is ~ 5 years ago and i'm not sure if people are still interested in the videos anymore. 
* Join us in Facebook, Google and Github. We are not maintaining the groups on these platforms anymore. Might as well ✂️ and feature Telegram channel more. 

Cutting many things might make the website feel empty, but instead we can feature whats left stronger. Lesser things is better than more but outdated information. Whats left that we can focus stronger are Meetups, Telegram Channel, Jobs and Companies, Visit Singapore section. 

### For now, new features are out of scope. For example:
* feature our meetup videos in the website. Likely have to work with engineers.sg to see if there is anyway we can embed the videos in https://ruby.sg but still drive traffic to https://engineers.sg

**Comment by adriangohjw** [comment]
What about embedding talks from Behind The Gems?

**Comment by TayKangSheng** [comment]
I think that is a good idea, but out of scope for this issue. Let me create another issue for this. One thing I want to make sure is that people know that the videos are hosted on engineers.sg.

**Comment by TayKangSheng** [comment]
Dug up some context about the current website.

The call for proposal of logo
https://github.com/rubysg/rubysg-reboot/issues/1

The voting of the logo 
https://github.com/rubysg/rubysg-reboot/issues/98

The redesign of the current website
https://github.com/rubysg/rubysg-reboot/issues/103
